tests/config/raw_non_overlap.yaml

```diff
@@ -0,0 +1,18 @@
+api-version: 0
+disklabel: gpt
+
+raw:
+  - input-offset: 0
+    output-offset: 33KiB
+    input:
+      uri: file://root.ext4
```

Comment:
We should not encourage writing ext4 images using the "raw" directive. Maybe add a small (e.g. 16KiB?) binary file with random bytes instead that can be used for such testing purposes.

Reply:
Switch to a new file called `test_file.bin`.

Reply:
Could you rename it to `random.bin`, so it is easily visible what the file contains?